backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -60,10 +60,13 @@
       <%= f.error_message_on :password %>
     <% end %>
 
-    <%= f.field_container :password do %>
-      <%= f.label :password_confirmation %>
-      <%= f.password_field :password_confirmation, :class => 'fullwidth' %>
-      <%= f.error_message_on :password_confirmation %>
+
+    <% if f.object.respond_to?(:password_confirmation) %>
+      <%= f.field_container :password do %>
```

Comment:
not sure, but I didn't change that at all in this PR, all it does is guard existing template code in a check. If that line ought to be changed, different PR I think?